lib/components/user/nav-login-button.tsx

```diff
@@ -75,7 +76,6 @@ const NavLoginButton = ({
               />
             </span>
           }
-          pullRight
         >
           <li className="header">{displayedName}</li>
```

Comment:
When you're adding the `<ul>` to this `Dropdown,`  I would make this one a more traditional header (not an `li`) and insert the `<ul>` underneath! This technically is not a part of the list, it's labeling the list.